### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NovemberTang**
## What does this change?

Update readme to reflect single-project nature of tool. Also, clarify running instructions

### CODE REVIEW

**Review by tjsilver — COMMENTED** [review]
I wonder if it might be good to move the usage to the top, and local dev/release to the bottom, so that users can easily get what they need?

File: README.md [hunk 1]
```diff
@@ -73,6 +83,12 @@ go run main.go S3.8 -profile <PROFILE> -region <REGION> [OPTIONAL_FLAGS]
 
 ## EC2.2
 
+### Usage
+
+```bash
+fsbp-fix EC2.2 -profile <PROFILE> -region <REGION> [OPTIONAL_FLAGS]
```
Comment (tjsilver) [inline README.md:89]: Does EC2.2 / S3.8 need to be capitalised?

Comment (NovemberTang) [reply README.md:89]: probably not, though I think that's beyond the scope of this PR as it'd be a breaking change for the tool

**Review by tjsilver — APPROVED** [review]
👍

**Review by tjsilver — APPROVED** [review]
Nice!

**Pull request merged**